### PR TITLE
SW-77512: add informational message for signaling

### PR DESCRIPTION
**Pull request by anstancu**
Closes: [SW-77512](https://drivenets.atlassian.net/browse/SW-77512) and [OTTC-215](https://drivenets.atlassian.net/browse/OTTC-215)

### CODE REVIEW

File: src/libcharon/sa/ikev2/tasks/dn_ike_signaling.c [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+METHOD(task_t, process_r, status_t,
+       private_dn_ike_sig_t *this, message_t *message)
+{
+    const uint64_t spi = message->get_initiator_spi(message);
+    notify_payload_t *notify = message->get_notify(message, INFORMATIONAL_ACTIVE_SIGNALING);
+
+    if (notify)
+    {
+        DBG0(DBG_IKE, "Received ACTIVATE request from 0x%lx", spi);
+        // TODO(lschlesinger) insert here call to zebra API
```
Comment (kfir-drivenets) [inline src/libcharon/sa/ikev2/tasks/dn_ike_signaling.c:58]: :+1: @lschlesinger-dn FYI

**Review by kfir-drivenets — DISMISSED** [review]
LGTM 💯

**Review by lbuga-dn — APPROVED** [review]

File: src/libcharon/plugins/vici/vici_control.c [hunk 2]
```diff
@@ -707,9 +707,11 @@ CALLBACK(activate_sa, vici_message_t*,
 		// 				and `standby` for all others
 		if (ike_id != ike_sa->get_unique_id(ike_sa))
 		{
+			ike_sa->activate_sa(ike_sa, FALSE, TRUE);
```
[the remark this answers is not in the document]
Comment (anstancu) [reply src/libcharon/plugins/vici/vici_control.c:708]: Done.

**Review by lbuga-dn — APPROVED** [review]

**Review by lbuga-dn — DISMISSED** [review]

File: src/libcharon/encoding/payloads/notify_payload.h [hunk 3]
```diff
@@ -153,6 +153,9 @@ enum notify_type_t {
 	FRAGMENTATION_SUPPORTED = 16430,
 	/* Signature Hash Algorithms, RFC 7427 */
 	SIGNATURE_HASH_ALGORITHMS = 16431,
+	/* DN specific INFORMATIONAL signaling messages */
+	INFORMATIONAL_ACTIVE_SIGNALING = 16432,
+	INFORMATIONAL_STANDBY_SIGNALING = 16433,
```
[the remark this answers is not in the document]
Comment (anstancu) [reply src/libcharon/encoding/payloads/notify_payload.h:158]: Done.

**Review by lbuga-dn — APPROVED** [review]

**Review by lschlesinger-dn — APPROVED** [review]

**Pull request merged**